### PR TITLE
Exchange trade

### PR DESCRIPTION
**Pull request by gyurka00**
Check trade history during the exchange tests

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit 2a2509cfd33943726dca09fb84f0477440233e95

https://app.netlify.com/sites/augmint/deploys/5ab52771df99536893cb05cf

File: cypress/integration/exchange.js [hunk 1]
```diff
@@ -172,7 +181,9 @@ describe("Augmint exchange", function() {
 
         cy.get("[data-testid=submitButton]").click();
 
-        cy.get("[data-testid=EthSubmissionSuccessPanel]").as("successPanel");
+        cy.get("[data-testid=EthSubmissionSuccessPanel]").as("successPanel").then(() => {
+            cy.get("[data-testid=tradeHistoryTable] table tbody").children().should('have.length', 5);
+        });
```
[the remark this answers is not in the document]
Comment (gyurka00) [reply cypress/integration/exchange.js]: Yes, it's true and I fixed it.

File: cypress/integration/exchange.js [hunk 2]
```diff
@@ -123,6 +129,8 @@ describe("Augmint exchange", function() {
                 cy.assertUserAEurBalanceOnUI(expectedAEurBalance);
                 cy.assertUserEthBalanceOnUI(expectedEthBalance);
 
+                cy.get("[data-testid=tradeHistoryTable] table tbody").children().should('have.length', 3);
```
[the remark this answers is not in the document]
Comment (gyurka00) [reply cypress/integration/exchange.js]: I made a refactor, but it was different than you wrote.
I would like to use globally the custom table template...

File: cypress/integration/exchange.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 describe("Augmint exchange", function() {
+    let tradeHistoryStartLength = document.querySelectorAll("[data-testid=trade-history] tbody tr").length;
```
Comment (szerintedmi) [inline cypress/integration/exchange.js]: if you save the tradehistory len here then the tests won't be independent. Ie. try running a last one adding `.only` (after it). Or try to swap the last two.
I would suggest to save it before each test where you need it or just place it in beforeEach.

Comment (gyurka00) [reply cypress/integration/exchange.js]: I made a huge refactor

**Pull request merged**